ush/soca/marine_recenter.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+import os
+from logging import getLogger
+from typing import Dict, List, Any
+from wxflow import AttrDict, FileHandler, logit, Task
+
+logger = getLogger(__name__.split('.')[-1])
+
+
+class MarineRecenter(Task):
+
+    @logit(logger, name="MarineRecenter")
+    def __init__(self, config):
+        logger.info("init")
+        super().__init__(config)
+
+        # Create a local dictionary that is repeatedly used across this class
+        local_dict = AttrDict(
+            {
+                "diags": os.path.join(self.runtime_config['DATA'], 'diags'),            # output dir for soca DA obs space
+                "obs_in": os.path.join(self.runtime_config['DATA'], 'obs') ,            # input      "           "
+                "bkg_dir": os.path.join(self.runtime_config['DATA'], 'bkg'),            # ice and ocean backgrounds
+                "anl_out": os.path.join(self.runtime_config['DATA'], 'Data'),           # output dir for soca DA
+                "static_ens": os.path.join(self.runtime_config['DATA'], 'static_ens')  # clim. ens.
+            }
+        )
+
+        # task_config is everything that this task should need
+        self.task_config = AttrDict(**self.config, **self.runtime_config, **local_dict)
+
+        #_soca_ensb_yaml_temp 
+        print(self.task_config) 
```

Comment:
use `logger.info` or `logger.debug`
Also, since printing a dictionary, please use `pformat`

Reply:
This is temporary for debugging